Sources/Logging/Destinations/Log+FileLogDestination.swift

```diff
@@ -2,30 +2,65 @@ import Foundation
 
 public extension Log {
 
+
+    /// An error produced by a `FileLogDestination`.
+    ///
+    /// - clearFailed: The log file couldn't be cleared (removed).
+    /// - itemFormatFailed: Failed to format a log item failed.
+    /// - itemWriteFailed: Writing a log item to file failed.
+    /// - metadataWriteFailed: Writing metadata message to file failed.
     public enum FileLogDestinationError: Error {
```

Comment:
I believe we should add a comment for each case instead of putting them all together in the enum documentation. It's what Apple does, and it allows one to consult the description of each case (from anywhere in the code). Also, it should prove to be easier to maintain in the long term whenever we rename/remove cases.

Reply:
You're right. I did this because Xcode auto generates it like this, but to use alt click on the cases themselves we have to define it on each one. 😉